trace/Span.md

```diff
@@ -1,8 +1,11 @@
-# Span API
+# Span
 
-This class represents a Span. A span contains a SpanContext and  allows users to record 
-tracing events based on the data model defined [here](https://github.com/census-instrumentation/opencensus-proto/blob/master/trace/trace.proto).
-All Span implementations MUST not allow users to read the recorded data, 
+Span represents a single operation within a trace. Spans can be nested to form a trace tree. 
```

Comment:
We talk about a "trace" here, but it's not given a definition, and more importantly, there's no class or object in the code that directly maps to a "trace".  Can you add a "Trace" heading above "Span" (or maybe in the README?) that says this?